server/manager/src/main/java/org/apache/accumulo/manager/TabletGroupWatcher.java

```diff
@@ -488,9 +488,17 @@ private TableMgmtStats manageTablets(Iterator<TabletManagement> iter,
       if (mtiError != null) {
         // An error happened on the TabletServer in the TabletManagementIterator
         // when trying to process this extent.
+        KeyExtent ke = null;
+        try {
+          ke = tm.getExtent();
+        } catch (IllegalStateException e) {
+          // thrown when no prev endrow in tablet metadata.
+          // suppress this exception and leave ke set to null
```

Comment:
could log this exception at trace

Reply:
Do you think logging this will add confusion in the log. There is a log message two lines down with the error from the server side.